app/controllers/articles_controller.rb

```diff
@@ -24,10 +35,10 @@ def feed
     end
 
     set_surrogate_key_header "feed"
+    set_cache_control_headers(10.minutes.to_i, stale_while_revalidate: 30, stale_if_error: 1.day.to_i)
 
-    set_cache_control_headers(600,
-                              stale_while_revalidate: 30,
-                              stale_if_error: 86_400)
+    @allowed_tags = FEED_ALLOWED_TAGS
```

Comment:
Since we already have a `render` right below, maybe pass them in as `locals`?

Reply:
done :) https://github.com/thepracticaldev/dev.to/pull/8046/commits/f775af85e1ac8422e11198be0aef1f019ad2635f